README.md

```diff
@@ -315,6 +315,18 @@ setVerbose(True)
 # turn off verbose mode
 setVerbose(False)
 ```
+### Bypassing merge task 
+
+Skipping the merge task is useful when the tasks results don't need to be merge into a list. To bypass the merge task, you can pass the *enableMerge* flag to the foreach object:
+
+```R
+# Enable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = TRUE))
+
+# Disable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = FALSE))
+```
+Note: User defined functions for the merge task is on our list of features that we are planning on doing.

```

Comment:
Instead of enableMerge, should we call it enableCombine? That would be more in line with the existing foreach api:

foreach(i = 1:1000, .combine = rbind, .options.azure = opt) {}